taccsite_cms/contrib/taccsite_sysmon/static/taccsite_sysmon/js/sysmon.js

```diff
@@ -0,0 +1,168 @@
+// GH-295: Use server-side logic instead of client-side
+
```

Comment:
__Note:__ This code is a polished version of the script in https://github.com/TACC/Core-CMS-Resources/pull/67/files, which is deleted in [`taccsite_custom` change](https://github.com/TACC/Core-CMS/pull/297/files#diff-00db495ec685cbdb37dc9dfede576323eb49fd5d4156be453590c5b7650bb9e6).